src/Components/Pet.js

```diff
@@ -1,9 +1,66 @@
 import React from 'react';
 import '../Styles/Pet.scss';
+import Header from './Header';
+
+//Posting happens before querying
+//Then query
+//Then data flows down from the query.
+
+//This is pulling the information posted from the 
+//On click of card, 
+//Pet ID from pet card
+//Pulls data to this component?
+//Or graphQL query can pull the info we need into this component?
 
 const Pet = () => {
   return (
-    <h1>Pet</h1>
+    <div className='pet-page-holder'>
+      <Header />
+      <div className='pet-card-container'>
+        <section className="single-pet-details-container">
+          <div className='about-owner-nav'>
+            <h2>Read More About the Owner >></h2>
+          </div>
+          <div className='middle-o-card-container'>
+            <div className='pet-details'>
+              <div className='light-pet-details'>
+                <h2 className='single-pet-name'>Stewart</h2>
+                <p className='single-pet-age'>9 years old</p>
+                <p className='single-pet-gender'>Female</p>
+                <p className='single-pet-type'>French Bull Dog</p>
+              </div>
+              <div className='single-pet-description'>
+                <p>Hi, I'm Stewy! I'm so happy to see you! I really like warm sweaters and belly rubs!
```

Comment:
All of this repetitive text is to assist with styling the page. We have to assume that there will be longer descriptions input by user's so we're accounting for that now with this hard-coded information.